packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -197,9 +205,9 @@ export interface IContextualMenuItem {
   href?: string;
 
   /**
-   * A collection of submenu items
+   * A collection of submenu items, or an IContextualMenuProps object to apply for the sub menu
    */
-  items?: IContextualMenuItem[];
+  items?: IContextualMenuItem[] | IContextualMenuProps;
 
```

Comment:
E.g. if you created an item that had both `items`, and `subMenuProps: { items: [] }` it's pretty obvious you're creating an ambiguous scenario for yourself. It's possible we can add warnings, but I think docs are probably sufficient.

Interested in if you still disagree and have some specific concerns! Are there cases we should consider?